addons/account/controllers/terms.py

```diff
@@ -4,10 +4,15 @@
 from odoo import http, _
 from odoo.http import request
 
+def sitemap_terms(env, rule, qs):
```

Comment:
Take it or leave it, you can ignore it as this linter error is officially not supported in our guidelines.
But since the file did not have a linter error before, would be cool to just add a blank line before and after this new method, avoiding 2 (unsupported) linter errors.
You decide on this one, I won't be upset if you ignore it ;-)